MinPlatformPkg/Include/Library/PlatformConfigCheckLib.h

```diff
@@ -0,0 +1,26 @@
+/** @file
+
+  Platform configuration check library for TestPointCheckLib
```

Comment:
I wouldn't assume this will always be linked against a TestPointCheckLib instance (if it actually does, that is fine).

```suggestion
  Platform configuration check library
```